docs/docs/command-line-interface.mdx

```diff
@@ -313,31 +313,32 @@ rasa data convert nlg --help
 
 ## rasa data validate
 
-You can check your domain, NLU data, or conversation data for mistakes and inconsistencies. 
+You can check your domain, NLU data, or story data for mistakes and inconsistencies. 
 To validate your data, run this command:
 
 ```bash
 rasa data validate
 ```
 
-By default, the validator searches only for errors in the data, e.g. the same training
-example being listed as an example for two intents.
-To catch minor issues that don't prevent training a model but might indicate messy data
-(e.g. unused intents), use the `--fail-on-warnings` flag.
+By default, the validator searches for errors in the data, e.g. two intents that have some
+identical training examples.
+Furthermore, the validator checks if you have any stories where different assistant actions follow from the same 
+dialogue history. Conflicts between stories will prevent a model from learning the correct
+pattern for a dialogue. 
+
+If you pass a `max_history` value to one or more policies in your `config.yml` file, provide the 
+smallest of those values in the validator command using the `--max-history <max_history>` flag. 
 
-You can also validate the story structure by running this command:
+You can also validate only the story structure by running this command:
 
 ```bash
 rasa data validate stories
 ```
 
-This validator checks if you have any stories where different assistant actions follow from the same 
-dialogue history. Conflicts between stories will prevent a model from learning the correct
-pattern for a dialogue. 
+Note: The `rasa data validate` and `rasa data validate stories` commands do **not** test if your [rules](./rules.mdx) 
```

Comment:
@JEM-Mosig could we also comfort the reader by telling them that rule-story consistency is being checked in the Core?